modules/mw.cx.MachineTranslationService.js

```
@@ -189,75 +189,69 @@ mw.cx.MachineTranslationService.prototype.cxtoken_error = function () {
+		.then((response) => response.json())
+		.catch(error => {
+			console.error('Error fetching mdwiki token:', error);
+		});
+
+	return result;
 };
 
 mw.cx.MachineTranslationService.prototype.fetchCXServerToken = function () {
-	// cxtoken
 
 	if (this.sourceLanguage === "mdwiki") {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)